OOP/task_04.py

```diff
@@ -0,0 +1,19 @@
+class Vehicle:
+    def __init__(self, make, model, year):
+        self.make = make
+        self.model = model
+        self.year = year
+    def show_info(self):
+        print(f"Make: {self.make} \nModel: {self.model} \nYear: {self.year}")
+    
+class ElectricCar (Vehicle):
+    def __init__(self, make, model, year, battery_size):
+        super().__init__(make, model, year)
+        self.battery_size = battery_size
+    def display_battery(self):
+        print(f"Battery Size: {self.battery_size}")
```

Comment:
Check what will happen if we do not include `super()` method call

```suggestion
    def __init__(self, make, model, year):
        self.make = make
        self.model = model
        self.year = year
        
    def show_info(self):
        print(f"Make: {self.make} \nModel: {self.model} \nYear: {self.year}")
    
    
class ElectricCar (Vehicle):
    def __init__(self, battery_size):
        self.battery_size = battery_size
        
    def display_battery(self):
        print(f"Battery Size: {self.battery_size}")

```

Reply:
If we don't include super(), the parent class (Vehicle) attributes won't be initialized, which may lead to missing attributes or errors when trying to access them